src/distributed/topology_worker.ts

```diff
@@ -99,6 +94,46 @@ export class TopologyWorker {
         this.coordinator.run();
     }
 
+    /** Thi smethod verifies tha t all topologies are running and properly registered */
+    private verifyTopologies(uuids: string[], callback: intf.SimpleCallback) {
```

Comment:
verifyTopologies does more than verify. A change of name to indicate that for start would be good. Reporting inconsistencies not just fixing would also be good.

Reply:
I agree. Will rename it.